app/src/main/res/layout/activity_main.xml

```diff
@@ -58,6 +58,7 @@
                 android:layout_width="match_parent"
                 android:layout_height="?attr/actionBarSize"
                 app:tabIndicator="@null"
+                app:tabGravity="fill"
```

Comment:
god, docs say *nothing* about it. What does it do? what's `tabGravity` even?

Reply:
I don't know. We had the same bug for top tabs and it was the same fix https://github.com/tuskyapp/Tusky/pull/239 (must have been one of my first PRs here :D)

Reply:
Ah, I understand the issue Noe that there's a screenshot! LGTM! (also, woah!)